### ISSUE TITLE
Implement CEA 608 parser

### ISSUE THREAD

**Issue by littlespex** [streaming-video-technology-alliance/common-media-library#62]
**Is your feature request related to a problem? Please describe.**
CEA 608/708 is currently implemented in dash.js, hls.js, and shaka-player based on code originally contributed to dash.js.

**Describe the solution you'd like**
Consolidate all of these implementations in to a single source in the Common Media Library

**Comment by avelad** [comment]
It would be nice if it also included CEA 708

**Comment by littlespex** [comment]
The title and description have been updated

**Comment by mpollingerQualabs** [comment]
For this ticket, we started with the implementation of the 608 parser. We took the implementation from the hls.js repo. This player does not have 708 parser but only 608.(*)
For the tests, we are going to take them from the Shaka player repo, as the other players don't have tests for this.

Also, Shaka has a implementation for the 708 parser which we are going to tackle after.



(*): taking a deeper look, it looks like the hls.js implementation of 608 has some parts of the 708 spec like changing background color and such. Still, we are not sure that it is a complete 708 spec implementation, hence the methods are called 608. Shaka actually has a parser for 608 and a parser for 708.
